gtwrap/matlab_wrapper/wrapper.py

```diff
@@ -7,15 +7,16 @@
 
 import os
 import os.path as osp
-import sys
 import textwrap
 from functools import partial, reduce
 from typing import Dict, Iterable, List, Union
 
+from loguru import logger
```

Comment:
new dependency?

Reply:
Loguru is a much better python logger. It's an existing dependency from when we refactored the python wrapper, so I decided to use it here as well instead of the `_debug` method that was defined.

Reply:
I see